packages/fast-jss-manager-angular/src/manage-jss.test.ts

```diff
@@ -0,0 +1,63 @@
+import { Component, Input, ElementRef } from "@angular/core";
+import { TestBed, async } from "@angular/core/testing";
+import manageJss, { ComponentStyles } from "./manage-jss";
+import { DesignSystemDirective } from "./design-system.directive";
+
+/**
+ * Component definition
+ */
+@Component({
+    selector: "example-with-directive",
+    template: `<design-system config="{color: 'red'}"><span [class]="className || ''">{{ text }}</span></design-system>`
+})
+
+/**
+ * Class definition
```

Comment:
This comment doesn't add any value.  Suggestion explaining the purpose of the class how/why it's necessary.